src/chapters/classes/methods.rst

```diff
@@ -106,20 +97,28 @@ an exercise.
            nextStudentId++;
        }
 
-       public void addGrade(int courseCredits, double grade) {
-           // Update the appropriate fields: numberOfCredits, gpa
-       }
-
        public String getGradeLevel() {
-           // Determine the grade level of the student based on numberOfCredits
+           if (this.numberOfCredits >= 0 && this.numberOfCredits < 11) {
```

Comment:
This is the solution for the first part of the exercises in the next chapter.  Could we use something like ``reportCredits`` or ``studentInfo`` that accomplishes a different task?